man/kernel-install.xml

```diff
@@ -158,8 +158,12 @@
 
   <refsect1>
     <title>Environment variables</title>
+
     <para>If <option>--verbose</option> is used, <varname>$KERNEL_INSTALL_VERBOSE=1</varname> will be set for
     the plugins. They may output additional logs in this case.</para>
+
+    <para>If <varname>MACHINE_ID=</varname> is set and nonempty, it will be used as <replaceable>MACHINE-ID</replaceable>,
+    overriding any automatic detection attempts.</para>
```

Comment:
maybe say that it expects a 32 character hex string

Reply:
Done